### PR TITLE
Let installer support 'format:f2fs' and add delay back to tpm

### PR DESCRIPTION
**Pull request by ceiba1985**
1. installer and fastboot may use installer.cmd file simultaneously. For format command, fastboot support creating file system by using 'format:fs_type', such as 'format:f2fs'. But installer doesn't support this syntax, since it only supports 2 arguments for format command currently.
Add support for format type syntax in installer so file system can be formatted based on the fs_type provided.
2. Add delay back to some functions of tpm, since the removement of delay may result in abnormal behavior of tpm operations, such as timeout for read/write operations.

Tracked-On: OAM-113370

### CODE REVIEW

File: libkernelflinger/lib.c [hunk 1]
```diff
@@ -1298,6 +1298,10 @@ VOID pause(UINTN seconds)
         uefi_call_wrapper(BS->Stall, 1, seconds * 1000000);
 }
 
+VOID pause_us(UINTN microseconds)
+{
+        uefi_call_wrapper(BS->Stall, 1, microseconds);
```
Comment (GangSecurity) [inline libkernelflinger/lib.c:1303]: It is always return true. NO delay here

Comment (James-Zzz) [reply libkernelflinger/lib.c:1303]: If we are using EDK2, Stall function is supported.
However, uefiwrappe does not support bs_stall actually, see implementation here:
http://inlubt0232.iind.intel.com:8080/source/xref/CELADON_S_MR0_MASTER/hardware/intel/efiwrapper/libefiwrapper/bs.c?r=79fb203a#245
So we need to implement pause ourselves, perhaps you can use 8253/8254 ?

**Review by James-Zzz — COMMENTED** [review]
Suggest to divide this commit into 2 commits, since they are not relevant.

**Review by GangSecurity — APPROVED** [review]

**Review by James-Zzz — APPROVED** [review]
LGTM

**Review by yang8621 — APPROVED** [review]

**Review by kwang13 — APPROVED** [review]

**Review by JeevakaPrabu — APPROVED** [review]

**Review by kwang13 — APPROVED** [review]

**Pull request merged**